docker/bookkeeper/entrypoint.sh

```diff
@@ -9,6 +9,9 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 
+ZK_HOME=/opt/zk
+BK_HOME=/opt/bk_all
```

Comment:
I'm not sure why we had all in the path, I guess it is because it was supposed to host all directories for all versions, but if we are collapsing all into one, then we should remove all. we should say `/opt/bookkeeper` rather than `/opt/bk`, though.